app/main.py

```diff
@@ -1,8 +1,45 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({self.name: self})
+
+    def add_wife(self, wife: object) -> None:
+        if wife:
+            self.wife = wife
+
+    def add_husband(self, husband: object) -> None:
+        if husband:
+            self.husband = husband
+
+
+def create_person_list(people1: list) -> list:
+    person_list1 = []
+    for person in people1:
+        one_person = Person(person["name"], person["age"])
+        one_person.add_wife(person.get("wife", None))
+        one_person.add_husband(person.get("husband", None))
```

Comment:
We need to add an attribute if the key exists and not None, not need to use the method in this way. Methods/setters will be needed if you fields will be private or protected